src/components/tag-input/tag-input.component.ts

```diff
@@ -829,3 +829,11 @@ export interface TagApi<T = any> {
  * The function used to return custom class information, for use in `ngClass`.
  */
 export type TagClassFunction<T = any> = (tag: T, index: number, selected: boolean) => (string | string[] | Set<string>);
+
+export type TagInputDisplayFn<T> = (option: T) => string;

```

Comment:
I would suggest `TagInputDisplayFunction` to match the existing function type name.

Reply:
Thanks Donna, that should now be updated